ci/pkl-pipelines/pay-deploy/node-runner.pkl

```diff
@@ -0,0 +1,94 @@
+amends "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.4#/Pipeline.pkl"
+
+import "../common/PayResources.pkl"
+import "../common/pipeline_self_update.pkl"
+
+resources = new {
+  pipeline_self_update.PayPipelineSelfUpdateResource("pay-deploy/node-runner.pkl", "master")
+  new PayResources.PayGitHubResource {
+    name = "node-runner-src"
+    repoName = "pay-ci"
+    source {
+      branch = "master"
+      paths = new {
+        "ci/docker/node-runner/*"
+      }
+    }
+  }
+  new PayResources.PayGitHubResource {
```

Comment:
Would be good to use the shared resource which defines this https://github.com/alphagov/pay-ci/blob/master/ci/pkl-pipelines/common/shared_resources.pkl#L14

Reply:
ah good one 👍 , I think the challenge here is finding the pre-made shared resources sometimes, I had a cursory look but nothing jumped out.